.github/workflows/build_and_deploy.yml

```diff
@@ -10,6 +15,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Docker compose up
+      if: startsWith(github.ref, 'refs/tags/')
```

Comment:
Why are we only doing `--build` on tags?

Reply:
Because this does the publish part.

Reply:
Ah, perhaps we can rename `docker-compose.ci.yml` to `docker-compose.deploy.yml` to make this more explicit?